src/daemon/absorber.cc

```diff
@@ -45,7 +45,8 @@ bool Absorber::Initialize() {
     return false;
   }
 
-  if (conf->has_cache() && conf->cache().has_direct()) {
+  if (conf->has_cache() && conf->cache().has_direct() &&
+      conf->cache().direct()) {
```

Comment:
Now absorber puts to log this message if it contains field `cache.direct`, even if it equals `false`. In this case message makes no sence. With this change message appears only if `cache.direct` equals `true`(which can't be used on absorber).